docs/excel/custom-functions-overview.md

```diff
@@ -28,109 +34,87 @@ In the cloned sample repo, you’ll see the following files:
 
 - **./src/customfunctions.js**, which contains the custom function code (see the simple code example above for the `ADD42` function).
 - **./config/customfunctions.json**, which contains the registration JSON that tells Excel about your custom function. Registration makes your custom functions appear in the list of available functions displayed when a user types in a cell.
-- **./index.html**, which provides a &lt;Script&gt; reference to the JS file. This file does not display UI in Excel.
-- **./manifest.xml**, which tells Excel the location of the HTML, JavaScript, and JSON files; and also specifies a namespace for all the custom functions that are installed with the add-in.
+- **./index.html**, which provides a &lt;Script&gt; reference to the JS file.
+- **./manifest.xml**, which tells Excel the location of the HTML, JavaScript, and JSON files; it also specifies a namespace for all the custom functions that are installed with the add-in.
```

Comment:
Good idea on the table. 
The removal of and is a personal preference thing I guess? I think it is weird to start an independent clause/most sentences with and. Theoretically it is grammatical though. I can change back if you'd like.

Reply:
@mscharlock  Not sure I get you. It's not a personal preference to avoid run-on sentences. How about, you put a period there and start a new sentence?